lib/GitHub.js

```diff
@@ -91,11 +91,11 @@ class GitHub {
 
    /**
     * Create a new Search wrapper
-    * @param {string} query - the query to search for
+    * @param {Search.Params} searchParameters - the query and other search parameters
```

Comment:
Not sure if jsdoc understands how to reach into other files (Search.js). Let me know if this will be a problem.